pkg/image/prune/imagepruner.go

```diff
@@ -190,6 +190,7 @@ func addImageStreamsToGraph(g graph.Graph, streams *imageapi.ImageStreamList, al
 
 		glog.V(4).Infof("Adding ImageStream %s/%s to graph", stream.Namespace, stream.Name)
 		isNode := imagegraph.EnsureImageStreamNode(g, stream)
+		imagegraph.EnsureAllImageStreamTagsNode(g, stream)
```

Comment:
> Wouldn't it make more sense to make EnsureAllImageStreamTagsNode private and call it from within EnsureImageStreamNode?

I was going to, but then decided that since we don't have sufficient information to build the entire imagestreamtag object from the imagestream, I needed to leave it disconnected so that it was possible for someone to create the fully populated ISTNodes without any surprises like: "you must create nodes of type X before nodes of type Y".